packages/dd-trace/test/encode/0.5.spec.js

```diff
@@ -4,6 +4,12 @@ const msgpack = require('msgpack-lite')
 const codec = msgpack.createCodec({ int64: true })
 const id = require('../../src/id')
 
+function randString (length) {
+  return Array.from({ length }, () => {
```

Comment:
What's the reason behind using `Array.from({ length })` instead of `new Array(length)` I wonder ?

Reply:
`Array.from({ length }, map)` is a faster way to create and map an array than separately doing `new Array(length)` and `array.map(fn)`.